### PR TITLE
update babel-config

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,14 +1,19 @@
 {
   "presets": [
-    ["es2015", { "loose": true }],
-    "stage-1",
+    ["env", {
+      "targets": {
+        "ie": 9,
+        "uglify": true
```
Comment (taion) [inline .babelrc]: what's this do?

File: .babelrc [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+      }
+    }],
     "react"
   ],
   "plugins": [
+    "transform-class-properties",
+    "transform-object-rest-spread",
+    "transform-export-extensions",
```
Comment (taion) [inline .babelrc]: maybe just use stage 1?

File: package.json [hunk 3]
```diff
@@ -45,16 +45,16 @@
     "babel-cli": "^6.11.4",
     "babel-core": "^6.13.2",
     "babel-eslint": "^8.0.1",
-    "babel-loader": "^6.2.5",
+    "babel-loader": "6.0.0",
```
Comment (taion) [inline package.json]: what?

Comment (jquense) [reply package.json]: YARN

File: .babelrc (hunk 2 above)
Comment (jquense) [reply .babelrc]: stage 1 is less maintainable, plus adds everything 1 -4 negating most of the env preset

File: .babelrc (hunk 1 above)
Comment (jquense) [reply .babelrc]: ensures it compiles code down to at least what uglify can parse, e.g es5

Comment (taion) [reply .babelrc]: isn't that what no-options `env` does? just target ES5, like with `latest`? i would just do plain `env` then, with `loose`.

Comment (jquense) [reply .babelrc]: i'm copying the CRA config here, they generally have good reasons for the settings

Comment (taion) [reply .babelrc]: fair enough. let's use `loose` though?

Comment (jquense) [reply .babelrc]: good call

File: .babelrc [hunk 4]
```diff
@@ -1,14 +1,20 @@
 {
   "presets": [
-    ["es2015", { "loose": true }],
-    "stage-1",
+    ["env", {
+      "targets": {
+        "ie": 9,
+        "uglify": true,
+        "loose": true
```
Comment (taion) [inline .babelrc]: not a target

**Pull request merged**